### PR TITLE
Fixing OBJMON and adding EXMON

### PR DESCRIPTION
**Pull request by v-p-b**
This request fixes the issue described in https://github.com/tklengyel/drakvuf/issues/28 and adds a new plugin for exception monitoring. 

Feel free to reject if these two things should go to separate PRs.

### CODE REVIEW

File: src/drakvuf.c [hunk 1]
```diff
@@ -1,5 +1,4 @@
-/*********************IMPORTANT DRAKVUF LICENSE TERMS***********************
- *                                                                         *
+/*********************IMPORTANT DRAKVUF LICENSE TERMS*********************** *                                                                         *
```
Comment (tklengyel) [inline src/drakvuf.c]: Stray whitespace added.

File: src/plugins/exmon/exmon.c [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+static GSList *traps;
+static output_format_t format;
+
+static event_response_t cb(drakvuf_t drakvuf, drakvuf_trap_info_t *info) {
+
+    vmi_instance_t vmi = drakvuf_lock_and_get_vmi(drakvuf);
+    KTRAP_FRAME trap_frame;
+    reg_t exception_record, ptrap_frame, exception_code;
+
+    page_mode_t pm = vmi_get_page_mode(vmi);
```
Comment (tklengyel) [inline src/plugins/exmon/exmon.c]: If you don't expect the pagemode to change at runtime (you are passed the OS boot) you could just read the pagemode once in the plugin init and store in the a global.

File: src/plugins/exmon/exmon.c [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+    trap->name = "KiDispatchException";
+    trap->module = "ntoskrnl.exe";
+    trap->type = BREAKPOINT;
+    trap->cb = cb;
+
+    if (!trap->u2.rva) {
+        return 0;
+    }
+
+    traps = g_slist_prepend(traps, trap);
```
Comment (tklengyel) [inline src/plugins/exmon/exmon.c]: For a single trap you could just create a static drakvuf_trap_t as in https://github.com/tklengyel/drakvuf/blob/master/src/plugins/objmon/objmon.c#L180, then you don't have to keep track of the linked list. It's really just cosmetic stuff.

**Comment by tklengyel** [comment]
Overall this is excellent! The first community-created plugin =) The commit messages should be squashed into a single commit but otherwise it looks good ;)

**Comment by v-p-b** [comment]
Thanks :) I'll clean things up and integrate the plugin based on the new branch.

**Comment by tklengyel** [comment]
Closing this PR now that the other one got merged.

**Pull request closed without merging**